packages/espresso/android/src/main/java/androidx/test/espresso/flutter/EspressoFlutter.java

```diff
@@ -100,7 +99,6 @@ private WidgetInteraction(
      * @param widgetActions one or more actions that shall be performed. Cannot be {@code null}.
      * @return this interaction for further perform/verification calls.
      */
-    @ExperimentalTestApi()
```

Comment:
I am struggling to find the original documentation for this annotation. There is an identically named class in compose that is exposed but I think it is for compose related apis. 

This annotation either is marking something experimental from us which we dont need anymore or it was required as an opt in feature and all the features have graduated.